### PR TITLE
work

### PR DESCRIPTION
**Pull request by odscjames**

### CODE REVIEW

File: libcoveofds/geojson.py [hunk 1]
```diff
@@ -223,32 +224,86 @@ def _process_network(self, geojson_feature_node_or_span: dict) -> None:
...
+                        # Is it inconsistent with what we have seen before?
+                        if json_diff_function(
+                            self._networks[network_id]["network_data_original"], network
+                        ):
+                            # record error for later
+                            self._inconsistent_network_ids_seen.add(network_id)
+
+                            # Check references to phases in contracts are consistent with what we have seen before.
+                            # This will give a more specific error on the phase_id.
+                            # (However, 2 errors will be recorded - one for network_id and one for phrase_id!
```
Comment (Lathrisk) [inline libcoveofds/geojson.py:244]: I think this should be `phase_id`?

File: tests/test_pythonvalidate.py [hunk 2]
```diff
@@ -50,8 +50,6 @@ def test_basic_1(filename):
     with open(expected_filename) as fp:
         expected_data = json.load(fp)
 
-    print(expected_data)
```
Comment (Lathrisk) [inline tests/test_pythonvalidate.py:53]: Would it be just as well squashing this into the previous commit? No worries either way.

File: libcoveofds/geojson.py (hunk 1 above)
Comment (odscjames) [reply libcoveofds/geojson.py:244]: Ah phrase_id -> phase_id

File: tests/test_pythonvalidate.py (hunk 2 above)
Comment (odscjames) [reply tests/test_pythonvalidate.py:53]: They are both about tests but the previous commit is a about a different test, so thought it best to leave separate

File: libcoveofds/geojson.py (hunk 1 above)
Comment (odscjames) [reply libcoveofds/geojson.py]: added commit to fix, rebased so behind release commit

**Review by Lathrisk — APPROVED** [review]

**Pull request merged**